test/test_zstar_coordinate.jl

```diff
@@ -278,7 +278,7 @@ end
             # Instead of initializing with random velocities, infer them from a random initial streamfunction
             # to ensure the velocity field is divergence-free at initialization.
             ψ = Field{Center, Center, Center}(grid)
-            set!(ψ, rand(size(ψ)...))
+            set!(ψ, 10^6 * rand(size(ψ)...))
```

Comment:
perhaps add a comment next to it? why 1e6? (is it related to the typical area of grid cells?)

Reply:
I added a comment in [597ccee](https://github.com/CliMA/Oceananigans.jl/pull/4487/commits/597ccee6d00cc964ff20795bd4ccf3b24c1f8341)